lib/DxilRootSignature/DxilRootSignatureValidator.cpp

```diff
@@ -27,6 +27,7 @@
 #include <vector>
 #include <set>
 #include <iostream>
+#include <ios>
```

Comment:
Since this is intended to resolve the same issue as `<iostream>` above it, I would prefer that it replace it entirely.

Please do the same for the `<iostream>` include in dxc_batch.cpp